bin/foreman-proxy-certs-generate

```diff
@@ -21,13 +21,14 @@ class ForemanProxyCertsGenerate
   private
 
   def generate_answers_file
+    initial_org = YAML.load_file('/etc/foreman-installer/scenarios.d/katello-answers.yaml')
```

Comment:
A couple of things here - 

We shouldn't hard code katello-answers because of Satellite, we should read `/etc/foreman-installer/scenarios.d/last_scenario.yaml` which would be a symlink to the last sceario instead.


Tests are failing because this path won't exist on a check out of git.  So we need a lot better error handling:

- make sure the file exists and it has a value for initial_organization before reading it
- if it doesn't exist, provide a default value ("Default_Organization")